src/web/routes/entities/index.ts

```diff
@@ -23,6 +29,28 @@ type AssociateEntityResponse = Response<
 	{ connectInstallation: ConnectInstallation }
 >;
 
+export type DisassociateEntityRequestParams = Omit<
```

Comment:
Was curious what everyone's thoughts are on separating the exported types into a separate file?

Reply:
I like it, keeps the route definitions tidy. Particularly for this route which had a few type definitions. I went ahead and made the change - if we want to apply this to all routes I can make the change in a separate PR.